src/Templates/src/templates/maui-mobile/PageModels/ProjectListPageModel.cs

```diff
@@ -0,0 +1,40 @@
+#nullable disable
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using MauiApp._1.Data;
+using MauiApp._1.Models;
+using MauiApp._1.Services;
+
+namespace MauiApp._1.PageModels;
+
+public partial class ProjectListPageModel : ObservableObject
+{
+	[ObservableProperty] private List<Project> _projects;
+	private readonly ProjectRepository _projectRepository;
+	private readonly ModalErrorHandler _errorHandler;
+
+	public ProjectListPageModel(ProjectRepository projectRepository, ModalErrorHandler errorHandler)
+	{
+		_projectRepository = projectRepository;
+		_errorHandler = errorHandler;
+	}
+
+	[RelayCommand]
+	private async Task Appearing()
+	{
+		Projects = await _projectRepository.ListAsync();
+	}
+
+	[RelayCommand]
+	private Task NavigateToProject(Project project)
+		=> Shell.Current.GoToAsync($"project?id={project.ID}");
+
+	[RelayCommand]
+	private async Task AddProject()
+	{
+		var project = new Project();
+		Projects.Add(project);
```

Comment:
`Projects` is a `List<Project>` and doesn't implement `INCC`, so this `.Add` will be visible to `INPC`? I believe that `INPC` will just see if you change the hole collection, not if you add or remove items from it, not 100% sure

Reply:
I think this is out of sync from our latest changes